e2e/rdctl.e2e.spec.ts

```diff
@@ -203,13 +235,218 @@ test.describe('HTTP control interface', () => {
     expect(body).toContain('no settings specified in the request');
   });
 
+  test.describe('rdctl', () => {
+    test('should show settings and nil-update settings', async() => {
+      const { stdout, stderr } = await rdctl(['list-settings']);
+
+      expect(stderr).toEqual('');
+      expect(stdout).toMatch(/"kubernetes":/);
+      const settings = JSON.parse(stdout);
+
+      expect(['version', 'kubernetes', 'portForwarding', 'images', 'telemetry', 'updater', 'debug', 'pathManagementStrategy']).toMatchObject(Object.keys(settings));
+
+      const args = ['set', '--container-engine', settings.kubernetes.containerEngine,
+        `--kubernetes-enabled=${ settings.kubernetes.enabled ? 'true' : 'false' }`,
+        '--kubernetes-version', settings.kubernetes.version];
+      const result = await rdctl(args);
+
+      expect(result.stderr).toEqual('');
+      expect(result.stdout).toContain('Status: no changes necessary.');
+    });
+
+    test('set: complains when no args are given', async() => {
+      const { stdout, stderr } = await rdctl(['set']);
+
+      expect(stderr).toContain('Error: set command: no settings to change were given');
+      expect(stderr).toContain('Usage:');
+      expect(stdout).toEqual('');
+    });
+
+    test('set: complains when option value missing', async() => {
+      const { stdout, stderr } = await rdctl(['set', '--container-engine']);
+
+      expect(stderr).toContain('Error: flag needs an argument: --container-engine');
+      expect(stderr).toContain('Usage:');
+      expect(stdout).toContain('');
+    });
+
+    test('set: complains when non-boolean option value specified', async() => {
+      const { stdout, stderr } = await rdctl(['set', '--kubernetes-enabled=gorb']);
+
+      expect(stderr).toContain('Error: invalid argument "gorb" for "--kubernetes-enabled" flag: strconv.ParseBool: parsing "gorb": invalid syntax');
```

Comment:
Actually, I like that full line because it shows:
1. We're handling attempts to supply an invalid boolean value
2. We're just going with the default cobra/golang-flag library's error message rather than doing something friendlier. We should really revise the test with the message we'd rather see, and implement it.